CONTRIBUTING.md

```diff
@@ -664,6 +665,14 @@ canonical: /path/to/canonical/doc/
 canonical: /{{< latest "influxdb" "v2" >}}/path/to/canonical/doc/
 ```
 
+## v2 equivalent documentation
+To display a notice about and link to equivalent v2 pages for pages in the 1.x
```

Comment:
To display a notice on a 1.x page that links to an equivalent 2.0 page, add the following frontmatter to the 1.x page: